code/modules/mob/living/say.dm

```diff
@@ -245,6 +245,22 @@ proc/get_radio_key_from_channel(var/channel)
 	log_say("[name]/[key] : [message]")
 	return TRUE
 
+
+/proc/animate_speechbubble(image/I, list/show_to, duration)
+	var/matrix/M = matrix()
+	M.Scale(0,0)
+	I.transform = M
+	I.alpha = 0
+	for(var/client/C in show_to)
+		C.images += I
```

Comment:
@LudwigVonChesterfield This is only showing it to the people in the show to list. Not just any random person.